### PR TITLE
Feature/modal options

### PR DESCRIPTION
**Pull request by kre8sions**
**New in Storybook**

-  side-modals
-  highlights
-  capping

screen shot 2018-11-12 at 12 52 08 pm
screen shot 2018-11-12 at 12 51 40 pm
screen shot 2018-11-12 at 12 51 30 pm

### CODE REVIEW

File: components/Capping/Capping.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    } = this.props
+
+    const css = [
+      className,
+      styles.cap,
+      styles[`cap-${cap}`],
+    ]
+     return (
+     <div>
+       <button className={classnames(css)}>{children}</button>
```
Comment (jasoncypret) [inline components/Capping/Capping.jsx]: This should use the button component right @markiearnold

Comment (jasoncypret) [reply components/Capping/Capping.jsx]: On second thought I don't think a capping component even needs to exist, just add these new options to buttons. We would still need the story to display it

File: components/Capping/CappingStory.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function CappingStory(stories) {
+  stories.add("Capping",
+    () => {
+      let props = {
+        className: text("className", ""),
+        cap: select("cap",["left", "right", "top", "bottom",["none" ,null]],"left")
+      }
+      return (
+        <Capping {...props}>
```
Comment (jasoncypret) [inline components/Capping/CappingStory.jsx]: @markiearnold should it work like this?

File: components/Capping/styles.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//Capping
+$bend:4px;
```
Comment (jasoncypret) [inline components/Capping/styles.scss]: Use border radius variables

File: components/Capping/styles.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+$cap-bottom: 0 0 $bend $bend;
+$cap-left: $bend 0 0 $bend;
+
+$cap-options:
+  "top" $cap-top,
+  "right" $cap-right,
+  "bottom" $cap-bottom,
+  "left" $cap-left;
+
+@each $tuple in $cap-options {
```
Comment (jasoncypret) [inline components/Capping/styles.scss]: Whats tuple?

File: components/DOBInput/index.jsx [hunk 5]
```diff
@@ -23,7 +23,9 @@ const DOBInput = ({ className, preventMinor, isValidDate, viewDate, ...props })
   }
 
   return (
-    <DatePicker {...props} className={className} />
+    <DatePicker {...props}
+        className={className}
```
Comment (jasoncypret) [inline components/DOBInput/index.jsx]: Did you re-organize these or is that your IDE?

File: components/Highlight/HighlightStory.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  stories.add("Highlight",
+    () => {
+      let props = {
+        side: select("side",['left','right','top','bottom'], 'left'),
+        color: select('color',['power-navy','power-red','power-blue','power-gold','power-green','power-navy-darker','power-royal'],'power-blue'),
+      }
+      return (
+
+          <div>
+              <Highlight {...props}/>
```
Comment (jasoncypret) [inline components/Highlight/HighlightStory.jsx]: @markiearnold ?

File: components/SideModal/SideModal.jsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    } = this.props
+
+    const css =[
+      [`${side}Modal`],
+      [`${side}Modal-${sizeOption}`],
+      [`${side}Modal-collapse-${collapseSize}`],
+    ]
+    return (
+      <div>
+        <Modal
```
Comment (jasoncypret) [inline components/SideModal/SideModal.jsx]: Do we need to extend other modal props? cc: @markiearnold

File: components/Capping/styles.scss (hunk 4 above)
Comment (kre8sions) [reply components/Capping/styles.scss]: renamed this to cap

File: components/DOBInput/index.jsx (hunk 5 above)
Comment (kre8sions) [reply components/DOBInput/index.jsx]: I think that was my IDE I never opened up date picker

File: components/Capping/styles.scss (hunk 3 above)
Comment (kre8sions) [reply components/Capping/styles.scss]: I passed $btn-border-radius-base to $bend @jasoncypret

File: components/Highlight/HighlightStory.jsx (hunk 6 above)
Comment (kre8sions) [reply components/Highlight/HighlightStory.jsx]: Thanks @markiearnold

**Comment by jasoncypret** [comment]
Are you ready for another review @kre8sions

**Pull request merged**